FG-100EF.yaml

```diff
@@ -0,0 +1,64 @@
+---
+manufacturer: Fortinet
+model: FortiGate 100EF
+part_number: FG-100EF
+slug: fg-100ef
+is_full_depth: false
+u_height: 1

```

Comment:
Please add a comment in markdown format linking to the manufacturer provided datasheet if you can find it

Reply:
Not sure I did this part correctly. Could you verify?

Reply:
All good, I added it for you :)